File: examples/fib.jl
function main()

	z = begin
		m = 5
		n = 8
		m / n
	end

	x = [0, 1, 2, 3, 4]
	for i in x
		println(fib(i))
	end
end

function fib(n :: Int64) :: Int64
	if n == 0
		return 0
	elseif n == 1
		return 1
	else
		return fib(n - 2) + fib(n - 1)
	end
end

File: examples/fib.rs
fn main() {

	let mut z: f64 = {
	let mut m = 5.0;
	let mut n = 8.0;
		m / n
	};

	println!("{:?}", z);

	let mut x = [0, 1, 2, 3, 4];
	for i in x {
		println!("{:?}", fib(i));
	}
}

fn fib(n: u64) -> u64 {

	if n == 0 {
		return 0;
	} else if n == 1 {
		return 1;
	} else {
		return fib(n - 2) + fib(n - 1);
	}
}
